testing/features/header.feature

```diff
@@ -60,12 +60,12 @@ Feature: Header component
     Background:
       Given a Standard Header component is on the page
 
-    @small_screen @failing @NP-1480
+    @small_screen @failing @NP-1885
     Scenario: English Header has be expanded to show the full search bar
       Then I am able to expand the search bar
       And I am able to collapse the search bar
 
-    @small_screen @failing @NP-1480
+    @small_screen @failing @NP-1885
```

Comment:
we can change this tag to `@small_screen @future_release @v4.1.0+`

Reply:
It's going to be in the next release though so I'll change it to `4.0.4`